### ISSUE TITLE
Feature 

### ISSUE THREAD

**Issue by Asrani-Aman** [CircuitVerse/CircuitVerse#4688]
### Is your feature request related to a problem? Please describe

 when setting the colour of the LED output, users are required to manually type the desired colour, such as 'purple.' However, this process can feel cumbersome and unintuitive.
 
 

https://github.com/CircuitVerse/CircuitVerse/assets/96644946/b336e79a-09cd-4ab4-95bc-ed98da9efb83

### Describe the solution you'd like

To address this, I propose integrating a colour picker feature into CircuitVerse. By adding a colour picker tool, users will have the ability to visually select colours from a palette, providing a more intuitive and user-friendly experience.

Implement a colour picker tool within the LED output settings.
Allow users to select colours visually from a colour palette.
Display the selected colour in real-time.
Provide an option to manually input colour values for users who prefer typing.

Here is the Figma Design Link

https://www.figma.com/file/KyqBZdYLhvJvnq6na8BwWm/Untitled?type=design&node-id=0%3A1&mode=design&t=mqSi10BqRoyu1aNk-1

### Are you working on this?

Yes

**Comment by Asrani-Aman** [comment]
@tanmoysrt bhaiya @nitin10s bhaiya can you please review this issue

**Comment by tanmoysrt** [comment]
LGTM
WDYT @Arnabdaz @Prerna-0202  ?

**Comment by Asrani-Aman** [comment]
@PiyushXmishra yes, feature will applicable in both VariableLEDs and DigitalLEDs.

**Comment by tanmoysrt** [comment]
Hi @Asrani-Aman, you can start working on the feature

**Comment by Asrani-Aman** [comment]
@Grumpyyash anmoysrt bhaiya I Have make A Pull Request, can you please Review it
